src/oc/nrnmpiuse.h.in

```diff
@@ -34,9 +34,6 @@
 /* define BlueGene with checkpointing */
 #undef BLUEGENE_CHECKPOINT
 
-/* Define bits for BGPDMA & 1 (ISend) & 2 (DMA spike transfer) & 4 (DMA Record Replay */
-#undef BGPDMA
```

Comment:
this reminds me that we should check docs for such options and update then.

Reply:
ok

Reply:
I think I found all of them !